_posts/2017-02-28-ome-files-c-0-3-2.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: Release of OME Files C++ 0.3.2
+intro-blurp: The OME team is pleased to announce the release of OME Files C++ 0.3.2
+---
+OME Files is a reference implementation of the OME Data Model and OME-TIFF file format for the storage and interchange of biological imaging data and metadata.  It provides support for:
+
+-  the OME Data Model (model objects, metadata store, XSL transforms and XML validation)
+-  OME-TIFF reading
+-  OME-TIFF writing
+
+This release contains a single bugfix for the Minimal TIFF writer to align its setting of the PlanarConfiguration TIFF tag with the OME-TIFF writer.
+
+**Downloads**
+
+The software release is available at 
+
+[archived downloads](http://downloads.openmicroscopy.org/ome-files-cpp/0.3.2/)
```

Comment:
Same as previous, I'd get rid of a bunch of unnecessary new lines here and when referencing trello below